src/wallet/rpcdump.cpp

```diff
@@ -75,16 +75,17 @@ std::string DecodeDumpString(const std::string& str)
 
 UniValue importprivkey(const UniValue& params, bool fHelp)
 {
-    if (fHelp || params.size() < 1 || params.size() > 3)
+    if (fHelp || params.size() < 1 || params.size() > 4)
         throw std::runtime_error(
             "importprivkey \"pivxprivkey\" ( \"label\" rescan )\n"
```

Comment:
nit: add new `fStakingAddress` optional argument to this line

Reply:
yep. done